### PR TITLE
trips scaffolding

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
work branded trips page

## How to Test
- [ ] Visit /work/trips
- [ ] Click tabs and see routes and content change respectively with selected tab

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/pages/trips/Trips.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                  <NavLink tag={tag} to={to}>
+                    {title}
+                  </NavLink>
+                </NavItem>
+              ))}
+            </Nav>
+
+            <Container>
+              <Row>
+                <Col md={6}>
```
Comment (kcvan) [inline src/pages/trips/Trips.tsx]: `md="6"`

File: src/pages/trips/Trips.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                  title: 'Past',
+                },
+                {
+                  tag: RRNavLink,
+                  to: '/work/trips/cancelled',
+                  title: 'Cancelled',
+                },
+              ].map(({ title, tag, to }) => (
+                <NavItem key={to}>
+                  <NavLink tag={tag} to={to}>
```
Comment (kcvan) [inline src/pages/trips/Trips.tsx]: i think you can remove the `tag` attribute and just put `<RRNavLink />` as the tag

File: src/pages/trips/Trips.tsx (hunk 1 above)
Comment (jerminatorhits) [reply src/pages/trips/Trips.tsx]: 🙏

File: src/pages/trips/Trips.tsx (hunk 2 above)
Comment (jerminatorhits) [reply src/pages/trips/Trips.tsx]: good catch! thx

**Pull request merged**